airbyte-json-validation/src/main/java/io/airbyte/validation/json/JsonSchemaValidator.java

```diff
@@ -63,6 +66,51 @@ protected JsonSchemaValidator(URI baseUri) {
     this.baseUri = baseUri;
   }
 
+  /**
+   * Create and cache a schema validator for a particular schema. This validator is used when
+   * {@link #testInitializedSchema(String, JsonNode)} and
+   * {@link #ensureInitializedSchema(String, JsonNode)} is called.
+   */
+  public void initializeSchemaValidator(String schemaName, JsonNode schemaJson) {
+    schemaToValidators.put(schemaName, getSchemaValidator(schemaJson));
+  }
+
+  public boolean testInitializedSchema(String schemaName, JsonNode objectJson) {
```

Comment:
Can we throw a comment on here indicating what a `true` vs `false` response means?